apps/admission-webhook-k8s/admission-webhook.yaml

```diff
@@ -13,6 +13,8 @@ spec:
     metadata:
       labels:
         app: admission-webhook-k8s
+        dns-name: admission-webhook-svc
+        "spiffe.io/spiffe-id-webhook": "true"
```

Comment:
I believe that quotes are not needed here.
I think we had quotes on some old example (maybe spire used it in its example), and then we just copied spire label everywhere, preserving quotes.
There is no point spending time to remove them but I suggest we at least don't use them in new examples.

Reply:
Tested. Trying to apply this without quotes I see an error: `Error from server (BadRequest): error when creating "../my-deployments-k8s/examples/basic": Deployment in version "v1" cannot be handled as a Deployment: json: cannot unmarshal bool into Go struct field ObjectMeta.spec.template.metadata.labels of type string` Looks like they're needed here

Reply:
I believe it should work fine like this:
```yaml
spiffe.io/spiffe-id-webhook: "true"
```

Reply:
Thanks, this option works, fixed

Reply:
Could you also fix this in the clusterspiffeid-webhook-template.yaml file, please?

Reply:
Sure, fixed